src/testArrays/ArrayCopy.java

```diff
@@ -0,0 +1,24 @@
+package testArrays;
+
+import java.util.Arrays;
+
+public class ArrayCopy {
+    public static void main(String[] args) {
+        int[] first = {1, 5, 23, 11};
+        int[] second = {3, 12, 7, 8, 10, -4};
+        arraycopy(first, 1, second, 2, 2);
+        System.out.println(Arrays.toString(second));
```

Comment:
Please add test cases: 
* copy first element
* copy last element
* when you expect to have exceptions
* copy zero element from array with zero elements to array with zero elements